lib/webidl2.js

```diff
@@ -863,7 +910,7 @@
       }
     };
 
-    const enum_ = function(store) {
+    const enum_ = function (store) {
```

Comment:
nit: whitespace after `function`

Reply:
Oops, the VSCode formatter 😬

Reply:
We should add `.vstudio` file at the base... I'm also using code, and we should start formatting this stuff properly.

Reply:
Wait, it should be `function enum_` instead of `const enum_`. Omitted on the previous PR.